src/SFA.DAS.ApprenticeCommitments/Data/Models/Apprenticeship.cs

```diff
@@ -21,5 +23,32 @@ public Apprenticeship(CommitmentStatement apprenticeship)
 
         public ICollection<CommitmentStatement> CommitmentStatements { get; private set; }
             = new List<CommitmentStatement>();
+
+        public CommitmentStatement LatestCommitmentStatement
+            => CommitmentStatements.OrderByDescending(x => x.CommitmentsApprovedOn).FirstOrDefault();
+
```

Comment:
This concerns me, I think it will introduce concurrency issues (think it would be better to have a field in the apprenticeship record pointing to the latest statement

Reply:
I would like to add the ability to update an apprenticeship in this PR, and separately solve the concurrency concerns.

I have created [ticket 597](https://skillsfundingagency.atlassian.net/browse/CS-597) to track it.

Reply:
ok agreed